src/utils/daos.c

```diff
@@ -381,11 +383,21 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 			break;
 
 		case 'a':
+			if (ap->attrname_str != NULL) {
+				fprintf(stderr,
+					"only one attribute name is allowed\n");
+				D_GOTO(out_free, rc = RC_NO_HELP);
+			}
 			D_STRNDUP(ap->attrname_str, optarg, strlen(optarg));
 			if (ap->attrname_str == NULL)
 				D_GOTO(out_free, rc = RC_NO_HELP);
 			break;
 		case 'v':
+			if (ap->value_str != NULL) {
+				fprintf(stderr,
+					"only one attribute name is allowed\n");
```

Comment:
Forgot to s/attribute name/attribute value/ after copying? :)

Reply:
Sorry Li Wei, I don't get this , can you clarify this a little bit ? ;-)

Reply:
I was suspecting this comment should read "only one attribute value is allowed\n", no?

Reply:
Ah yes, for sure !!!!!